src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -504,10 +513,13 @@ RadixContainer<T> partition_by_radix(const RadixContainer<T>& radix_container,
   jobs.reserve(input_partition_count);
 
   for (ChunkID input_partition_idx{0}; input_partition_idx < input_partition_count; ++input_partition_idx) {
```

Comment:
Again not your code, but can you change the `input_partition_idx` to use `auto`?